web/source/osk/activeLayout.ts

```diff
@@ -1,14 +1,19 @@
 namespace com.keyman.osk {
   type KeyDistribution = text.KeyDistribution;
 
+
   export class ActiveKey implements LayoutKey {
 
+    static readonly DEFAULT_PAD=15;          // Padding to left of key, in virtual units
+    static readonly DEFAULT_RIGHT_MARGIN=15; // Padding to right of right-most key, in virtual units
+    static readonly DEFAULT_KEY_WIDTH=100;   // Width of a key, if not specified, in virtual units
```

Comment:
These constants make the code a lot clearer! 👍

Reply:
😸